cli/tests/test_diff_command.rs

```diff
@@ -3726,17 +3726,18 @@ fn test_diff_stat_rounding() {
     let work_dir = test_env.work_dir("repo");
     let output = work_dir.run_jj(["diff", "--stat"]);
     insta::assert_snapshot!(output, @r"
-    ... | 101 +-
-    ... | 301 +-
-    ... |  10 +
+    .. | 101 +-
```

Comment:
This changed due to small changes in the math, but it's testing we don't crash here so I don't think the difference matters.